utils/tests/utils.go

```diff
@@ -130,6 +141,21 @@ func compare(expected, actual []string) error {
 	return nil
 }
 
+// Actual list contains dir and files, so we check prefix (for dirs) or equality (for files)
```

Comment:
Let's try to improve this comment. I'm pretty sure it won't be clear for the random user.